.gitignore

```diff
@@ -2,3 +2,5 @@
 cc_code/bin/
 cc_code/build/
 python/**/__pycache__
+.idea/
```

Comment:
What's this? Just for my understanding, thanks

Reply:
This is just ignoring the config files from my IDE